Dependencies.md

```diff
@@ -13,10 +13,10 @@ This is mostly so that we keep up with dependencies for which we want new featur
 For example some dependencies that were in the middle of big updates were skipped as they didn't add anything of value at the time, but were being updated day after day seemingly fixing bugs added the previous day. As in that case that wasn't a dependency we wanted to be on top off - we skipped it.
 
 Through the rest of the development cycle dependencies shouldn't be updated unless:
+
 1. required for development
-2. a particular bug fix/feature is really important
-	
-	
+2. a particular bug fix/feature is really important.
+
```

Comment:
It's really preferable to not have any other but the necessary changes when a dependency is updated.

Did you make this change by accident? If so - I am fine with moving it to another commit in this PR.

Reply:
yes, `license/cla` stuck again (and it seems like I'm missing the access to re-run the webhook). I can forse push again with removing this change :see_no_evil:

Reply:
I still see changes on `Dependencies.md`

Reply:
Yeah, you are currently removing a line, @olegbespalov . Not a big problem, but still preferable if we don't do it in the commit updating a dependancy

Reply:
I've reverted it, using checkout now, should be no changes